xml/docu_styleguide.structure.xml

```diff
@@ -2076,6 +2380,12 @@ xml:id="tab-install-source"</screen>
   &lt;/listitem&gt;
  &lt;/varlistentry&gt;
 &lt;/variablelist&gt;</screen>
+      </example>
+      <example xml:id="ex-variable-list-source-adoc" outputformat="adoc">
+        <title>Example of a variable list (source)</title>
+<screen>This book consists of several parts:
+Installation:: Learn about the installation and initial configuration of a Linux system.
```

Comment:
Not sure, but doesn't ADoc require a linebreak? Perhaps it would be good practice to have one between a paragraph and starting a variable list like so:

```suggestion
<screen>This book consists of several parts:

Installation:: Learn about the installation and initial configuration of a Linux system.
```